Demos/Blazorise.Demo.Bootstrap.RC/Startup.cs

```diff
@@ -33,6 +34,9 @@ public void ConfigureServices( IServiceCollection services )
             services.AddServerSideBlazor().AddHubOptions( ( o ) =>
             {
                 o.MaximumReceiveMessageSize = 1024 * 1024 * 100;
+            } ).AddCircuitOptions( o =>
+            {
+                o.JSInteropDefaultCallTimeout = TimeSpan.FromSeconds( 5 );
```

Comment:
Maybe don't check this in. Its for testing only

Reply:
Yeah, I forgot to comment that will not be merged.